src/services/itemLogin/test/index.test.ts

```diff
@@ -532,36 +538,57 @@ describe('Item Login Tests', () => {
           it('Successfully create item login with username', async () => {
             assertIsDefined(anotherItem);
             const payload = USERNAME_LOGIN;
-            await saveItemLoginSchema({ item: anotherItem as unknown as DiscriminatedItem });
+            const { itemLoginSchema } = await saveItemLoginSchema({
+              item: anotherItem as unknown as DiscriminatedItem,
+            });
 
             const res = await app.inject({
               method: HttpMethod.Post,
               url: `${ITEMS_ROUTE_PREFIX}/${anotherItem.id}/login`,
               payload,
             });
 
-            expect(res.json().name).toEqual(payload.username);
             expect(res.statusCode).toBe(StatusCodes.OK);
+            const guestInDb = await rawItemLoginRepository.findBy({
+              name: payload.username,
+              itemLoginSchema,
+            });
+            expect(guestInDb).toHaveLength(1);
+            expect(guestInDb[0].lastAuthenticatedAt).toBeDefined();
+            expect(res.json().name).toEqual(payload.username);
           });
 
           it('Successfully reuse item login with username', async () => {
             assertIsDefined(anotherItem);
             const payload = USERNAME_LOGIN;
             // pre-create pseudonymized data
-            const { guest: m } = await saveItemLoginSchema({
+            const { guest, itemLoginSchema } = await saveItemLoginSchema({
               item: anotherItem as unknown as DiscriminatedItem,
               memberName: payload.username,
+              // purposely set lastAuthenticatedAt date to yesterday
+              lastAuthenticatedAt: new Date(new Date().getDate() - 1),
```

Comment:
This results in `Thu Jan 01 1970 01:00:00 GMT+0100 (Central European Standard Time)` which is not what you want I guess. 

Maybe using date-fns would be easier ?

Reply:
```suggestion
              lastAuthenticatedAt: new Date(new Date() - (24 * 60 * 60 * 1000)),
```